File: go-sdk/pkg/models/operations/suspenduser.go
package operations

type SuspendUserPathParams struct {
	ID int32 `pathParam:"style=simple,explode=false,name=id"`
}

type SuspendUserRequest struct {
	PathParams SuspendUserPathParams
}

type SuspendUserResponse struct {
	ContentType string
	StatusCode  int64
}

File: ts-sdk/src/sdk/models/operations/suspenduser.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class SuspendUserPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: number;
}


export class SuspendUserRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: SuspendUserPathParams;
}


export class SuspendUserResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
